rasa_sdk/tracing/instrumentation/attribute_extractors.py

```diff
@@ -56,3 +56,17 @@ def extract_attrs_for_validation_action(
         "slots_to_validate": json.dumps(list(slots_to_validate)),
         "action_name": self.name(),
     }
+
+
+def extract_attrs_for_create_api_response(
+    events: List[Dict[Text, Any]],
```

Comment:
Should we extract actually all the event type names? Also is there anything of use in `messages`, maybe utter labels if present? And/Or count of messages?